### PR TITLE
Migrate to zookeeper-less kafka 

### PR DESCRIPTION
**Pull request by hubertdeng123**
follow up to https://github.com/getsentry/self-hosted/pull/2445, which was reverted since it broke existing installs

### CODE REVIEW

File: install/turn-things-off.sh [hunk 1]
```diff
@@ -6,6 +6,10 @@ if [[ -n "$MINIMIZE_DOWNTIME" ]]; then
 else
   # Clean up old stuff and ensure nothing is working while we install/update
   $dc down -t $STOP_TIMEOUT --rmi local --remove-orphans
+  # TODO(getsentry/self-hosted#2489)
+  if docker volume ls | grep -qw sentry-zookeeper; then
+    docker volume rm sentry-zookeeper
```
Comment (BYK) [inline install/turn-things-off.sh:11]: Well, I love this that said if the upgrade fails for some reason you have now lost data?

Comment (hubertdeng123) [reply install/turn-things-off.sh]: Moved this removal to the end of the installation then to ensure it will only run if the upgrade succeeds.

**Comment by hubertdeng123** [comment]
I just tested this upgrade on our dogfood instance, and it seems to be running successfully

**Review by BYK — APPROVED** [review]
🚀

**Review by IanWoodard — APPROVED** [review]

**Pull request merged**